src/style.css

```diff
@@ -1,3 +1,166 @@
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --white: #fff;
+  --lightgrey: #f3f3f3;
+  --darkgrey: #060b35;
+  --grey: #616070;
+  --blue: #00acdc;
+}
+
+.card {
+  width: 200px;
+}
+
+.card__rectangle {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+
+  width: 198px;
+  flex-shrink: 0;
+
+  border-radius: 5px;
+  border: 1px solid var(--lightgrey);
+  background: var(--white);
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  flex-shrink: 0;
+  margin: 32px 19px 0;
+}
+
+.card__description {
+  color: var(--darkgrey);
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin: 40px 16px 0;
+}
+
+.card__code {
+  color: var(--grey);
+  font-size: 10px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 14px;
+  margin: 4px 82px 0 16px;
+}
+
+.card__price-bloc {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 24px 14px 0 16px; /*верхній був 56*/
+}
+
+.card__price-word {
+  display: inline-block;
+
+  color: var(--grey);
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+}
+
+.card__price-value {
+  display: inline-block;
+  color: var(--darkgrey);
+  text-align: right;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: var(--blue);
+  width: 166px;
+  height: 40px;
+  flex-shrink: 0;
+  border-radius: 5px;
+  margin: 16px;
+  border: 1px solid var(--blue);
+}
+
+.card__buy {
+  text-decoration: none;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  text-transform: uppercase;
+  color: var(--white);
+  padding: 11px 70px;
+}
+
+.rate {
+  display: flex;
+  /* position: absolute; */
+  top: 276px;
+  left: 16px;
+  width: 166px;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin: 16px 14px 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  flex-shrink: 0;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--4 :nth-child(-n + 4) {
```

Comment:
apply styles for all amount of stars to make your solution flexible

Reply:
I do not understand how these stars should behave when interacting with them, clarify the essence of the task, please

Reply:
so we should be able to change modifier from stars--4 to stars--3 and amount of active stars should be changed